[WEB-9711] Update Zendesk ticket retention policy language (#38471)
DataDog/documentation: content/en/getting_started/support/_index.md

When support in your preferred language is not available, you can continue working with Datadog Support in English.

## Ticket retention policy

> Our retention policy changed on June 12, 2026: closed tickets, including their attachments, are now deleted 15 months after their last update. Reach out to us with any questions.

Reviewer: Do we need to add the date to this?

```suggestion
Closed tickets, along with any attachments, are deleted 15 months after their last update.
```

Author: @estherk15 I do believe so, but also we can clarify with the stakeholder on the ticket: https://datadoghq.atlassian.net/browse/WEB-9711

Reviewer: It's not a blocker, but if it does need to be added back in, you can prepend the sentence with the date:

```suggestion
As of June 12, 2026, closed tickets, along with any attachments, are deleted 15 months after their last update.
```
